js/renovation/ui/scheduler/appointment/allDayStrategy/types.ts

```diff
@@ -0,0 +1,31 @@
+enum EAllDayPanelModeOptionTypes {
+  all = 'all',
+  allDay = 'allDay',
+  hidden = 'hidden',
+}
+
+type AllDayPanelModeType =
+  EAllDayPanelModeOptionTypes.all
+  | EAllDayPanelModeOptionTypes.allDay
+  | EAllDayPanelModeOptionTypes.hidden;
+
+enum EAllDayAppointmentStrategy {
```

Comment:
Do we realy need a prefix before enums (E-)? :)
This practice is normal for the interface.